wave_lang/kernel/wave/wave.py

```diff
@@ -708,12 +709,15 @@ def compile_to_mlir(
                 self.constraints,
                 options,
                 self.grid_type.dims,
+                entrypoint_name,
             )
             with mb.module_op.context, Location.unknown():
-                module_op = Module.create()
+                module_op = builtin_d.ModuleOp(loc=Location.unknown(context))
```

Comment:
Nit: you shouldn't need `loc=` inside the `Location` context manager. If it doesn't work, file a bug for upstream.